### DISCUSSION TITLE
Move all the windows of the same app

### DISCUSSION THREAD

**Discussion by junsukha** [rxhanson/Rectangle#1357] [Q&A]
Hi,
is there a feature that move all the windows of the same app to another monitor?
For example, I have multiples of windows of VScode open on a monitor A and would like to move all the windows to a monitor B.
As of now, I have to move each window separately.
Any updates since #326?

Edit
Guess this feature is only available on Pro version? #923

**Comment by rxhanson** [comment]
Indeed this is in [Rectangle Pro](https://rectangleapp.com/pro) (paid) and not in Rectangle. At this time I don't have plans to add it into Rectangle.
